### DISCUSSION TITLE
I get this error I added Git LFHS and set it up to track the yolo file mentioned in the trace but I still get this error not sure what to do.

### DISCUSSION THREAD

**Discussion by Sami-Krothapalli** [community/community#143540] [Programming Help]
### Body

remote: Resolving deltas: 100% (2/2), done.
remote: error: Trace: 2e3951b75dd58964eddc03b17073dff2cf611ba5056041d64ca7b0e6ff3be1ff
remote: error: See https://gh.io/lfs for more information.
remote: error: File yolov8x.pt is 130.55 MB; this exceeds GitHub's file size limit of 100.00 MB
remote: error: GH001: Large files detected. You may want to try Git Large File Storage - https://git-lfs.github.com.
To https://github.com/Sami-Krothapalli/AI-Football-Analysis-System.git

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by Nathanael349** [comment]
Even after setting up Git LFS to track yolov8x.pt, you're still encountering the error because the large file was already committed to your repository's history before Git LFS was configured. To resolve this, you need to remove yolov8x.pt from your Git history and then re-add it so that Git LFS can track it properly. You can do this by running:

1. Remove the file from the repository's history: `git rm --cached yolov8x.pt`
2. Add Git LFS tracking: `git lfs track "yolov8x.pt"`
3. Add the changes: `git add .gitattributes yolov8x.pt`
4. Commit the changes: `git commit -m "Track yolov8x.pt with Git LFS"`
5. Push to GitHub: `git push origin main`

This should solve your issue!
